### ISSUE TITLE
drawing scene graphs -- KeyError: 'chair'

### ISSUE THREAD

**Issue by kevinstan** [ashual/scene_generation#11]
Hi, thanks for your great work and releasing the code! I am trying to reproduce the results from Fig. 3 using the following script:

`python ~/scene_generation/scripts/sample_images.py \
        --checkpoint models/128/checkpoint_with_model.pt \
        --use_gt_boxes 1 \
        --use_gt_masks 1 \
        --use_gt_attr 1 \
        --use_gt_textures 1 \
        --save_gt_imgs 1 \
        --save_graphs 1 \
        --save_layout 1 \
        --output_dir samples
`

but it seems like 'chair' is not in the vocabulary:

```
Loading model from  models/128/checkpoint_with_model.pt
include other:  False
called set_image_size (128, 128)
/vision/u/kevintan/anaconda3/envs/scene_generation/lib/python3.6/site-packages/torch/nn/functional.py:2693: UserWarning: Default grid_sample and affine_grid behavior will be changed to align_corners=False from 1.4.0. See the documentation of grid_sample for details.
  warnings.warn("Default grid_sample and affine_grid behavior will be changed "
Lossy conversion from float32 to uint8. Range [0.0, 255.0]. Convert image to uint8 prior to saving to suppress this warning.
Lossy conversion from float32 to uint8. Range [16.71304702758789, 251.0]. Convert image to uint8 prior to saving to suppress this warning.
Lossy conversion from float32 to uint8. Range [0.0, 255.0]. Convert image to uint8 prior to saving to suppress this warning.
Traceback (most recent call last):
  File "/vision/u/kevintan/scene_generation/scripts/sample_images.py", line 312, in <module>
    run_model(args, checkpoint, args.output_dir)
  File "/vision/u/kevintan/scene_generation/scripts/sample_images.py", line 289, in run_model
    graph_img = draw_scene_graph(objs[i], triples[i], vocab)
  File "/vision/u/kevintan/scene_generation/scene_generation/vis.py", line 160, in draw_scene_graph
    objs_list.append(vocab['object_to_idx'][vocab['object_idx_to_name'][objs[i].item()]])
KeyError: 'chair'
```

Any help would be greatly appreciated. 

Thanks,
Kevin

**Comment by ashual** [comment]
I've fixed this bug, thanks for telling me.
Pay attention that there is no need to use both `--use_gt_boxes` and `--use_gt_attr` because if you use the real GT boxes the model will ignore the size and location attributes...

**Comment by kevinstan** [comment]
Thank you @ashual, it works now. For future reference: Fixed at e4b98ea437329b1a51deb67811e2b6593baa8ff6
